sessionctx/variable/tidb_vars.go

```diff
@@ -257,6 +257,14 @@ const (
 
 	// TiDBUseAlloc indicates whether the last statement used chunk alloc
 	TiDBUseAlloc = "last_sql_use_alloc"
+
+	// TiDBInefficientScanRatio indicates whether to print the query as slow query if mvcc scanned versions/
```

Comment:
```suggestion
	// TiDBInefficientScanRatio indicates to print the query as slow query if mvcc scanned versions/
```

Is the word *whether* unnecessary here?